streampipes-rest-extensions/src/main/java/org/apache/streampipes/rest/extensions/migration/DataSinkMigrationResource.java

```diff
@@ -47,7 +45,6 @@ public class DataSinkMigrationResource extends MigrateExtensionsResource<
   @POST
   @Consumes(MediaType.APPLICATION_JSON)
   @JacksonSerialized
-  @PreAuthorize(AuthConstants.IS_ADMIN_ROLE)
```

Comment:
Is this check redundant or why do we want to remove it?

Reply:
I removed this for now because this functionality required a dependency to `streampipes-rest` in the  `streampipes-rest-extensions` module, which pulled in all core dependencies in the extension modules, adding a lot of overhead.
Since there's currently no authorization for extensions in place, we can address this later if needed.